app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] @Elhambasir Greta job implementing the changes requested, however in Like model you have stated that it belongs to the user instead of author, to be inline with your schema, it produces error below when I try to create a like
![image](https://github.com/Elhambasir/blog-app/assets/61408860/ad355a63-6b6c-4502-85d6-d897c7a6acdd)

Reply:
Fixed